src/vue/forms/BuyAtomicSwapForm.vue

```diff
@@ -138,15 +172,31 @@ export default {
       vuexTypes.assetByCode,
       vuexTypes.statsQuoteAsset,
     ]),
-    totalPrice () {
-      return MathUtil.multiply(this.atomicSwapAsk.price, this.form.amount || 0)
+
+    isDiscountExist () {
+      return Number(this.discount)
```

Comment:
Why you return a number?

Reply:
I think method with 'is' prefix must return boolean

Reply:
changed